tests/test_hydra.py

```diff
@@ -1033,8 +1033,15 @@ def test_override_with_invalid_group_choice(
     assert re.search(msg, str(e.value)) is not None
 
 
-@mark.parametrize("config_path", ["dir1", "dir2"])
-@mark.parametrize("config_name", ["cfg1", "cfg2"])
+@mark.parametrize(
+    "config_path",
+    [
+        "dir1",
+        "dir2",
+        os.path.abspath("tests/test_apps/app_with_multiple_config_dirs/dir2"),
+    ],
+)
+@mark.parametrize("config_name", ["cfg1", "cfg2", "cfg2"])
```

Comment:
```suggestion
@mark.parametrize("config_name", ["cfg1", "cfg2"])
```
There's duplication here.